bin/version

```diff
@@ -1 +1 @@
-0.45.2.1
+1.45.2.1
```

Comment:
Well that's a rather large jump. Was that a typo?

Reply:
Not a typo.  They keep the same semantic versioning between the enterprise and non-enterprise JARs, but the enterprise begins with `1.0xxx` and the non-enterprise begins with `0.0.xxx`.  🤷